polaris-tokens/package.json

```diff
@@ -2,13 +2,35 @@
   "name": "@shopify/polaris-tokens",
   "version": "5.0.0-rc.3",
   "description": "",
-  "main": "dist/index.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "browser": "dist/browser/index.js",
```

Comment:
Lets match this to previous tokens

Reply:
Roger roger, removed the `browser` build target: https://github.com/Shopify/polaris/pull/5601/commits/78712ee13c7439a201dced7649dd9ca643f13a53